lib/db/mysql/index.js

```diff
@@ -27,6 +33,30 @@ function MysqlStore(options) {
     return next();
   };
   this._pool = mysql.createPool(options);
+
+  // Enforce sane defaults on every new connection.
+  // These *should* be set by the database by default, but it's nice
+  // to have an additional layer of protection here.
+  this._pool.on('connection', function(connection) {
```

Comment:
We could set a flag, like `connection._fxa_strict = true`, to only do it once per connection.